src/lib/init.js

```diff
@@ -49,18 +49,36 @@ export function init(configUpdates) {
 
 				// set cookies on digitrust domain after consent submitted
 				const { addEventListener, getVendorConsents } = cmp.commands;
+				if (config.digitrust.pixel) {
+					addEventListener('consentStringUpdated', digitrustPixel);
+				}
 				if (config.digitrust.redirects) {
 					addEventListener('consentStringUpdated', digitrustRedirect);
 				}
 
-				function digitrustRedirect() {
-					getVendorConsents([64], (result) => {
-						if (result && result.vendorConsents && result.vendorConsents[64]) {
-							window.location.replace(`${config.digitrustRedirectUrl}${encodeURIComponent(window.location.href)}`);
+				function getDigiTrustVendorConsents(success, failure) {
```

Comment:
I think that this function will be more readable and mainainable if it's written in the following way:

```
const ifVendorIdIsSet = (id, getVendorConsents, callback) => () => {
    getVendorConsents(id, (result) => {
        get(result, `vendorConsent.${id}`) && isFunction(callback) && callback()
    })
}

const sendInvisiblePixel = () => {...}

const onConsentStringUpdated = ifVendorIdIsSet(
    metadata.digitrustVendorId,
    getVendorConsents
    sendInvisiblePixel)

addEventListener('consentStringUpdated', onConsentStringUpdated)
```